leftwm-watchdog/src/utils.rs

```diff
@@ -0,0 +1,315 @@
+use std::{
+    collections::HashSet,
+    fs,
+    path::{Path, PathBuf},
+    process::{Child, Command},
+    sync::{atomic::AtomicBool, Arc},
+};
+
+use xdg::BaseDirectories;
+
+pub const fn get_help_template() -> &'static str {
+    "\
+{name} {version}
+{author-with-newline}{about-with-newline}
+{usage-heading} {usage}
+
+{all-args}
+"
+}
```

Comment:
I forgot to mention, the leftwm(bin) wasn't completely independent from the utils. These two functions should be the only duplicated code. I think 15 lines is worth the trade-off of (presumably) better compile times.